### DISCUSSION TITLE
"Your focus-trap must have at least one container with at least one tabbable node in it at all times" when opening modal

### DISCUSSION THREAD

**Discussion by royvdmaas** [laravel/nova-issues#6327] [Q&A]
Dear community,

I am using Nova 4, which I have upgraded from 3 following the upgrade guide in the Nova Documentation. 
Whenever I try to open a modal (doesn't matter if it's from a custom action or a standard delete confirmation), I get the gray overlay, and the modal doesn't show. The console shows the following: "Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times", originating from Modal.vue, line 125.

I cannot find out why it's doing this. It happens on my local environment, as well as a staging server. 

Things I've tried: 
- php artisan view:clear
- php artisan cache:clear
- recompiling assets with npm
- npm and composer update

I hope someone can point me in the right direction.

Thanks,

Roy

**Comment by crynobone** [comment]
Unable to reproduce the issue, please provide full reproducing repository based on fresh installation as suggested in the bug report template (or you can refer to https://github.com/nova-issues for example)

**Reply by royvdmaas** [reply]
Thank you for your response. 
I have done the following:
- Pruned my composer.json and package.json files for unused or obsolete packages
- removed both the vendor and node_modules directories.
- Reran composer install and npm install so all my packages are fresh
- Compiled all my assets with npm and still the issue persisted.

As mentioned earlier, this occurs on both my local install as well as my staging server.

Here are my composer.json and package.json files:
[composer.json](https://github.com/laravel/nova-issues/files/14881864/composer.json)
[package.json](https://github.com/laravel/nova-issues/files/14881865/package.json)

Many thanks in advance.

**Reply by royvdmaas** [reply]
PS: The error occurs on line 125 of webpack://laravel/nova/resources/js/components/Modals/Modal.vue

**Reply by crynobone** [reply]
Please provide full reproduction repository as this can't be replicated in our dusk tests

**Reply by royvdmaas** [reply]
I'm sorry, but I am not allowed to publish the code in the repository. 
Is there a way to disable tabbable/focus-trap in Nova, be it for testing? Because all my modals are currently broken. I am however not sure if there might be an underlying problem that causes this error.

**Reply by crynobone** [reply]
> I'm sorry, but I am not allowed to publish the code in the repository.

I'm asking for a reproducing repository and it is not the actual product code. We have multiple dusk tests including what's you reporting here and tests still are successful.
